db/buckets.go

```diff
@@ -22,6 +22,8 @@ const (
 	TransactionsByBlockNumberAndIndex       // maps block number and index to transaction
 	ReceiptsByBlockNumberAndIndex           // maps block number and index to transaction receipt
 	StateUpdatesByBlockNumber
+	ClassesRootKey
+	ClassesTrie
```

Comment:
I don't see why we need both of these. I believe `ClassesTrie` without any suffix is sufficient to store `ClassesRootKey`. The same applies to StateRootKey and StateTrie.

Reply:
Yeap, done